java/libphonenumber/src/com/google/i18n/phonenumbers/nano/Phonemetadata.java

```diff
@@ -2,6 +2,8 @@
 
 package com.google.i18n.phonenumbers.nano;
 
+// @@protoc_insertion_point(imports_scope)
```

Comment:
Ah, that's good (that it was auto-generated) :)

I'll patch it in and see if I get the same output.